src/main/kotlin/de/docs_as_co/intellij/plugin/drawio/editor/DrawioWebView.kt

```diff
@@ -34,8 +36,21 @@ class DrawioWebView(lifetime: Lifetime) : BaseDrawioWebView(lifetime) {
         }
     }
 
+    override fun handleResponse(response: IncomingMessage.Response) {
+        when(response) {
+            is IncomingMessage.Response.Export -> {
+                val payload = response.data.split(",")[1]
+                val decodedBytes = Base64.getDecoder().decode(payload)
+                val svg = String(decodedBytes)
+                _xmlContent.set(svg)
+            }
+        }
+    }
```

Comment:
This should be removed and handled at the calling side of the request.

Reply:
I think this is fixed by now